### PR TITLE
Remove checks for git and local path from pipfile

### PR DESCRIPTION
**Pull request by shreekarSS**
Signed-off-by: Shreekara <sshreeka@redhat.com>

## Related Issues and Dependencies
Fixes: https://github.com/thoth-station/kebechet/issues/1125
…

### CODE REVIEW

**Comment by KPostOffice** [comment]
To give some context. @shreekarSS and I talked a bit about this and realized that the result from the function raising the issues in git repositories wasn't being used. This was the only reference to the code in the repo, so we decided it could be removed.  The return statement to the get direct dependency version so that when old dependencies and new dependencies are calculated they don't throw error and instead report no differences in reported version.

File: kebechet/managers/update/update.py [hunk 1]
```diff
@@ -235,62 +235,6 @@ def _get_direct_dependencies_requirements(req_dev: bool) -> set:
...
-            # TODO: open a PR to fix this
-            raise DependencyManagementError(
-                f"Failed to load Pipfile.lock file: {str(exc)}"
-            ) from exc
-
-        result = {}
-
-        for package_name, package_info in pipfile_lock_content["default"].items():
-            if "git" in package_info:
-                self._create_unsupported_package_issue(package_name, "git")
```
Comment (codificat) [inline kebechet/managers/update/update.py:253]: I think the definition of this method should also be removed with the proposed change, as it is only used within the removed code

https://github.com/thoth-station/kebechet/blob/a7c2f9d6c207cb3740b268fd2502c0bdb6c15bdf/kebechet/managers/update/update.py#L294-L327)

File: kebechet/managers/update/update.py [hunk 2]
```diff
@@ -235,62 +235,6 @@ def _get_direct_dependencies_requirements(req_dev: bool) -> set:
...
-                    "Failed to find version for package that uses local installation."
-                )
-            result[package_name.lower()] = {
-                "dev": False,
-                "version": package_info["version"][len("==") :],
-            }
-        # Close git as a source issues.
-
-        issue = self.get_issue_by_title(
-            _ISSUE_UNSUPPORTED_PACKAGE.format(env_name=self.runtime_environment)
```
Comment (codificat) [inline kebechet/managers/update/update.py:285]: The definition of that message should also be removed if it is not used anymore

https://github.com/thoth-station/kebechet/blob/a7c2f9d6c207cb3740b268fd2502c0bdb6c15bdf/kebechet/managers/update/update.py#L77-L80

**Review by codificat — CHANGES_REQUESTED** [review]
Thanks for this!

I believe there are a couple of additional code snippets that should be cleaned up as part of this change

File: kebechet/managers/update/update.py (hunk 1 above)
Comment (shreekarSS) [reply kebechet/managers/update/update.py:253]: ack, thanks

File: kebechet/managers/update/update.py (hunk 2 above)
Comment (shreekarSS) [reply kebechet/managers/update/update.py:285]: ack, thanks

**Comment by codificat** [comment]
/lgtm
/approve

Thanks!

**Pull request merged**